.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+# workflow name
+name: Generate release-artifacts
+
+# on events
+on:
+  push:
+    tags:
+        - '*'
+
+# workflow tasks
+jobs:
+  generate:
+    name: Generate cross-platform builds
+    runs-on: ubuntu-latest
+    steps:
+      - uses: olegtarasov/get-tag@v2.1
+        id: tagName
+        with:
+          tagRegex: "v(.*)"  # Optional. Returns specified group text as tag name. Full tag string is returned if regex is not defined.
+          tagRegexGroup: 1 # Optional. Default is 1.
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Generate build files
+        uses: thatisuday/go-cross-build@v1.1.0
+        with:
+            platforms: 'linux/amd64'
+            package: 'src'
+            name: 'foreman_ygg_worker-${{ steps.tagName.outputs.tag }}'
+            compress: 'true'
+            dest: 'dist'
+      - name: Generate distribution tarball
+        run: |
+          make distribution-tarball
+          sudo mv *.tar.gz dist/
```

Comment:
Why do you need sudo for this?

Reply:
`Generate build files` step creates the `dist` directory and sets it to be owned by root  with 755 permissions :/

Reply:
Does `go-cross-build` build as root then? Perhaps since https://github.com/thatisuday/go-cross-build/blob/master/Dockerfile doesn't have a `USER` option?

Reply:
> Does go-cross-build build as root then?

Most likely. Not sure if there's any sane way to make the uid/gid match both inside and outside the container, so even if the dockerfile specified a user, it might not really help us.